js/client/modules/@arangodb/testing.js

```diff
@@ -121,6 +122,7 @@ let optionsDocumentation = [
 ];
 
 const optionsDefaults = {
+  'dumpAgencyOnError': false,
```

Comment:
```suggestion
  'dumpAgencyOnError': true,
```

Reply:
Let's flip the default here, as MaxN needs the dump.
Everyone can still turn off the dumps locally if required.